aerospike-mixin/dashboards/aerospike-instance-overview.libsonnet

```diff
@@ -0,0 +1,964 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'aerospike-instance-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+local lokiDatasourceName = 'loki_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local lokiDatasource = {
+  uid: '${%s}' % lokiDatasourceName,
+};
+
+local unavailablePartitionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(instance, aerospike_cluster, job) (aerospike_namespace_unavailable_partitions{job=~"$job", aerospike_cluster=~"$aerospike_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{instance}}',
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Unavailable partitions',
+  description: 'Number of unavailable data partitions in an Aerospike node.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 1,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    text: {
+      valueSize: 100,
+    },
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-59542pre',
```

Comment:
is this needed?

Reply:
This appears in 32 files across `jsonnet-libs`, probably added during dashboard conversion.

Reply:
This seems to appear pretty regularly across `jsonnet-libs`